### ISSUE TITLE
Named Entity Recognition Doesnt work

### ISSUE THREAD

**Issue by ElToro13** [hankcs/HanLP#1398]
I am running this project in Google Colab with GPU, which I run this command
recognizer = hanlp.load(hanlp.pretrained.ner.CONLL03_NER_BERT_BASE_UNCASED_EN)

It gives following error,
Downloading https://file.hankcs.com/hanlp/ner/ner_conll03_bert_base_uncased_en_20200104_194352.zip to /root/.hanlp/ner/ner_conll03_bert_base_uncased_en_20200104_194352.zip
92.72%, 356.6 MB/384.6 MB, 7.3 MB/s, ETA 4 s      Executing op Range in device /job:localhost/replica:0/task:0/device:CPU:0
Executing op Cast in device /job:localhost/replica:0/task:0/device:CPU:0
Executing op HashTableV2 in device /job:localhost/replica:0/task:0/device:CPU:0
Executing op LookupTableImportV2 in device /job:localhost/replica:0/task:0/device:CPU:0
Executing op HashTableV2 in device /job:localhost/replica:0/task:0/device:CPU:0
Executing op LookupTableImportV2 in device /job:localhost/replica:0/task:0/device:CPU:0
Downloading https://storage.googleapis.com/bert_models/2018_10_18/uncased_L-12_H-768_A-12.zip to /root/.hanlp/thirdparty/storage.googleapis.com/bert_models/2018_10_18/uncased_L-12_H-768_A-12.zip
100.00%, 388.8 MB/388.8 MB, 7.9 MB/s, ETA 0 s      
Extracting /root/.hanlp/thirdparty/storage.googleapis.com/bert_models/2018_10_18/uncased_L-12_H-768_A-12.zip to /root/.hanlp/thirdparty/storage.googleapis.com/bert_models/2018_10_18
Executing op RandomUniform in device /job:localhost/replica:0/task:0/device:CPU:0
Executing op Sub in device /job:localhost/replica:0/task:0/device:CPU:0
Executing op Mul in device /job:localhost/replica:0/task:0/device:CPU:0
Executing op Add in device /job:localhost/replica:0/task:0/device:CPU:0
Executing op VarHandleOp in device /job:localhost/replica:0/task:0/device:CPU:0
Executing op VarIsInitializedOp in device /job:localhost/replica:0/task:0/device:CPU:0
Executing op LogicalNot in device /job:localhost/replica:0/task:0/device:CPU:0
Executing op Assert in device /job:localhost/replica:0/task:0/device:CPU:0
Executing op AssignVariableOp in device /job:localhost/replica:0/task:0/device:CPU:0
Executing op VarHandleOp in device /job:localhost/replica:0/task:0/device:CPU:0
Executing op TruncatedNormal in device /job:localhost/replica:0/task:0/device:CPU:0
Executing op VarHandleOp in device /job:localhost/replica:0/task:0/device:CPU:0
Executing op Fill in device /job:localhost/replica:0/task:0/device:CPU:0
Executing op VarHandleOp in device /job:localhost/replica:0/task:0/device:CPU:0
Executing op VarHandleOp in device /job:localhost/replica:0/task:0/device:CPU:0
Executing op VarHandleOp in device /job:localhost/replica:0/task:0/device:CPU:0
Executing op VarHandleOp in device /job:localhost/replica:0/task:0/device:CPU:0
Executing op VarHandleOp in device /job:localhost/replica:0/task:0/device:CPU:0
Executing op VarHandleOp in device /job:localhost/replica:0/task:0/device:CPU:0
Executing op VarHandleOp in device /job:localhost/replica:0/task:0/device:CPU:0

Failed to load https://file.hankcs.com/hanlp/ner/ner_conll03_bert_base_uncased_en_20200104_194352.zip. See stack trace below
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/hanlp/utils/component_util.py", line 43, in load_from_meta_file
    obj.load(save_dir, **load_kwargs)
  File "/usr/local/lib/python3.6/dist-packages/hanlp/common/component.py", line 244, in load
    self.build(**merge_dict(self.config, training=False, logger=logger, **kwargs, overwrite=True, inplace=True))
  File "/usr/local/lib/python3.6/dist-packages/hanlp/common/component.py", line 255, in build
    loss=kwargs.get('loss', None)))
  File "/usr/local/lib/python3.6/dist-packages/hanlp/components/taggers/transformers/transformer_tagger.py", line 35, in build_model
    model, tokenizer = build_transformer(transformer, max_seq_length, len(self.transform.tag_vocab), tagging=True)
  File "/usr/local/lib/python3.6/dist-packages/hanlp/layers/transformers/loader.py", line 108, in build_transformer
    with stdout_redirected(to=os.devnull):
  File "/usr/lib/python3.6/contextlib.py", line 81, in __enter__
    return next(self.gen)
  File "/usr/local/lib/python3.6/dist-packages/hanlp/utils/io_util.py", line 520, in stdout_redirected
    stdout_fd = fileno(stdout)
  File "/usr/local/lib/python3.6/dist-packages/hanlp/utils/io_util.py", line 501, in fileno
    fd = getattr(file_or_fd, 'fileno', lambda: file_or_fd)()
io.UnsupportedOperation: fileno
https://file.hankcs.com/hanlp/ner/ner_conll03_bert_base_uncased_en_20200104_194352.zip was created with hanlp-2.0.0-alpha.5, while you are running 2.0.0-alpha.20. Try to upgrade hanlp with
pip install --upgrade hanlp

---------------------------------------------------------------------------

UnsupportedOperation                      Traceback (most recent call last)

/usr/local/lib/python3.6/dist-packages/hanlp/utils/component_util.py in load_from_meta_file(save_dir, meta_filename, transform_only, load_kwargs, **kwargs)
     42                     load_kwargs = {}
---> 43                 obj.load(save_dir, **load_kwargs)
     44             obj.meta['load_path'] = load_path

9 frames

UnsupportedOperation: fileno

During handling of the above exception, another exception occurred:

NameError                                 Traceback (most recent call last)

/usr/local/lib/python3.6/dist-packages/hanlp/utils/component_util.py in load_from_meta_file(save_dir, meta_filename, transform_only, load_kwargs, **kwargs)
     54                 f'Try to upgrade hanlp with\n'
     55                 f'pip install --upgrade hanlp')
---> 56         exit(1)
     57 
     58 

NameError: name 'exit' is not defined

Any help will be appreciated!
Thank in advance

**Comment by hankcs** [comment]
[auto-reply] Thanks for your comment. However, the essential information is required. Please carefully fill out the form then reopen it.

**Comment by hankcs** [comment]
Seems like Google Colab changed the IO object which breaks the redirection. I'll look into this.

**Comment by yilu1015** [comment]
Any updates on this? I have the same issue. Tried `pip install -U hanlp` but didn't work either.

```
TypeError: __init__() got an unexpected keyword argument 'spm_model_file'
https://file.hankcs.com/hanlp/ner/ner_bert_base_msra_20200104_185735.zip was created with hanlp-2.0.0-alpha.5, while you are running 2.0.0-alpha.29. Try to upgrade hanlp with
pip install --upgrade hanlp
```

```
TypeError: __init__() got an unexpected keyword argument 'spm_model_file'

During handling of the above exception, another exception occurred:

NameError                                 Traceback (most recent call last)
<ipython-input-2-1583550f92bd> in <module>
----> 1 recognizer = hanlp.load(hanlp.pretrained.ner.MSRA_NER_BERT_BASE_ZH)

/usr/local/lib/python3.7/site-packages/hanlp/__init__.py in load(save_dir, meta_filename, transform_only, load_kwargs, **kwargs)
     49     save_dir = hanlp.pretrained.ALL.get(save_dir, save_dir)
     50     from hanlp.utils.component_util import load_from_meta_file
---> 51     return load_from_meta_file(save_dir, meta_filename, transform_only=transform_only, load_kwargs=load_kwargs, **kwargs)
     52 
     53 

/usr/local/lib/python3.7/site-packages/hanlp/utils/component_util.py in load_from_meta_file(save_dir, meta_filename, transform_only, load_kwargs, **kwargs)
     54                 f'Try to upgrade hanlp with\n'
     55                 f'pip install --upgrade hanlp')
---> 56         exit(1)
     57 
     58 

NameError: name 'exit' is not defined
```

**Comment by hankcs** [comment]
> Any updates on this? I have the same issue. Tried `pip install -U hanlp` but didn't work either. Using Python 3.7 now.
> 
> ```
> TypeError: __init__() got an unexpected keyword argument 'spm_model_file'
> https://file.hankcs.com/hanlp/ner/ner_bert_base_msra_20200104_185735.zip was created with hanlp-2.0.0-alpha.5, while you are running 2.0.0-alpha.29. Try to upgrade hanlp with
> pip install --upgrade hanlp
> ```
> 
> ```
> TypeError: __init__() got an unexpected keyword argument 'spm_model_file'
> 
> During handling of the above exception, another exception occurred:
> 
> NameError                                 Traceback (most recent call last)
> <ipython-input-2-1583550f92bd> in <module>
> ----> 1 recognizer = hanlp.load(hanlp.pretrained.ner.MSRA_NER_BERT_BASE_ZH)
> 
> /usr/local/lib/python3.7/site-packages/hanlp/__init__.py in load(save_dir, meta_filename, transform_only, load_kwargs, **kwargs)
>      49     save_dir = hanlp.pretrained.ALL.get(save_dir, save_dir)
>      50     from hanlp.utils.component_util import load_from_meta_file
> ---> 51     return load_from_meta_file(save_dir, meta_filename, transform_only=transform_only, load_kwargs=load_kwargs, **kwargs)
>      52 
>      53 
> 
> /usr/local/lib/python3.7/site-packages/hanlp/utils/component_util.py in load_from_meta_file(save_dir, meta_filename, transform_only, load_kwargs, **kwargs)
>      54                 f'Try to upgrade hanlp with\n'
>      55                 f'pip install --upgrade hanlp')
> ---> 56         exit(1)
>      57 
>      58 
> 
> NameError: name 'exit' is not defined
> ```

Happy new year. I've released 2.0.0a30 just now could you test it out?

**Comment by yilu1015** [comment]
Just upgraded to  2.0.0a30. Everything is working properly now. Thanks again, and happy lunar new year!
